account_bank_statement_import_camt_details/models/camt_parser.py

```diff
@@ -41,6 +41,13 @@ def parse_transaction_details(self, ns, node, transaction):
                 './ns:TwnNm', namespaces={'ns': ns})
             if city_node:
                 address_values['city'] = city_node[0].text
+            generic_nodes = address_node[0].xpath(
+                './ns:AdrLine', namespaces={'ns': ns})
+            address_values.update({
+                'address_line_' + str(i): node.text
+                for i, node in enumerate(generic_nodes)
+                if generic_nodes
+            })
```

Comment:
Final suggestion: it would be very easy to add test coverage for your work by just changing [these two lines](https://github.com/OCA/bank-statement-import/blob/10.0/account_bank_statement_import_camt/test_files/test-camt053#L265) to `<AdrLine>` fields instead of StrtNm and TwnNm.